### DISCUSSION TITLE
Support for SLXD?

### DISCUSSION THREAD

**Discussion by J-Lindkvist** [karlcswanson/micboard#30] [Q&A]
Hello

will slxd work in micboard in the future?
Now then it works in Shure WB

Best Regards

**Comment by karlcswanson** [comment]
Hello @J-Lindkvist,
The Shure command protocol is similar across devices. It would be easy to add support, but I currently do not have access to SLX-D hardware for development and testing.

**Reply by benji-york** [reply]
If you were lent SLX-D hardware (I don't have any, just asking in case someone else likes the idea), is that something you would have time to work on?

**Reply by J-Lindkvist** [reply]
If I had lived in the United States, I would have lent you a system.

**Reply by karlcswanson** [reply]
If someone were to ship me a device, I'd be happy to add support and provide return shipping.  The [command language](https://pubs.shure.com/command-strings/SLXD/en-US) is most similar to the Axient Digital line.

**Comment by creativedamages** [comment]
If someone has one and teamviewer would that save it being sent?

**Reply by J-Lindkvist** [reply]
I could create a small network with an SLXD receiver and a VPN server, if that would be helpful? or if it is better with a computer (Win10 / MacOSx) and SLXD receiver and TeamViewer

**Reply by karlcswanson** [reply]
I appreciate the offers! There's a lot of turning mics on and off.  Power-cycling the receivers, reseating network cables, and tapping on the mic to make sure everything works.  I'd need it on my desk for a bit to get support added in.

If this were more than a little side project, I'd obviously rent/buy the right equipment for development and testing.  Trying to not loose too much on this (though the SLX-D looks very affordable).

**Comment by s0600204** [comment]
Just a quick note to reference related issue #47
